blogposts/announcing-sourcegraph-3.10.md

```diff
@@ -0,0 +1,198 @@
+- Authorization provider configuration errors in external services will be shown as site alerts. [#6061](https://github.com/sourcegraph/sourcegraph/issues/6061)
+
+### Removed
+
+## 3.9.4
+
+### Changed
+
+- The experimental search pagination API's `PageInfo` object now returns a `String` instead of an `ID` for its `endCursor`, and likewise for the `after` search field. Experimental paginated search API users may need to update their usages to replace `ID` cursor types with `String` ones.
+
+### Fixed
+
+- The experimental search pagination API no longer omits a single repository worth of results at the end of the result set. [#6286](https://github.com/sourcegraph/sourcegraph/issues/6286)
+- The experimental search pagination API no longer produces search cursors that can get "stuck". [#6287](https://github.com/sourcegraph/sourcegraph/issues/6287)
+- In literal search mode, searching for quoted strings now works as expected. [#6255](https://github.com/sourcegraph/sourcegraph/issues/6255)
+- In literal search mode, quoted field values now work as expected. [#6271](https://github.com/sourcegraph/sourcegraph/pull/6271)
+- `type:path` search queries now correctly work in indexed search again. [#6220](https://github.com/sourcegraph/sourcegraph/issues/6220)
+
+## 3.9.3
+
+### Changed
+
+- Sourcegraph is now built using Go 1.13.3 [#6200](https://github.com/sourcegraph/sourcegraph/pull/6200).
+
+## 3.9.2
+
+### Fixed
+
+- URI-decode the username, password, and pathname when constructing Postgres connection paramers in lsif-server [#6174](https://github.com/sourcegraph/sourcegraph/pull/6174). Fixes a crashing lsif-server process for users with passwords containing special characters.
+
+## 3.9.1
+
+### Changed
+
+- Reverted [#6094](https://github.com/sourcegraph/sourcegraph/pull/6094) because it introduced a minor security hole involving only Grafana.
+  [#6075](https://github.com/sourcegraph/sourcegraph/issues/6075) will be fixed with a different approach.
+
+The [changelog for this and previous releases](https://github.com/sourcegraph/sourcegraph/blob/master/CHANGELOG.md#3.10) is available on GitHub.
+
+## Thank you
+
+Thank you to the many people who contributed to Sourcegraph since the last release!
+
+- [@prayashm](https://github.com/prayashm)
+- [@abeyerpath](https://github.com/abeyerpath)
+- [@gerbal](https://github.com/gerbal)
+- [@jlangston](https://github.com/jlangston)
+- [@425183525](https://github.com/425183525)
+- [@kevachen](https://github.com/kevachen)
+- [@brentshermana](https://github.com/brentshermana)
+
+**Deploy or upgrade:** [Local](https://docs.sourcegraph.com/#quickstart-guide) | [AWS](https://github.com/sourcegraph/deploy-sourcegraph-aws) | [DigitalOcean](https://marketplace.digitalocean.com/apps/sourcegraph?action=deploy&refcode=48dfb3ccb51c) | [Kubernetes cluster](https://github.com/sourcegraph/deploy-sourcegraph)
```

Comment:
We are getting rid of the DigitalOcean one click app for now so we should remove the link to it here.

https://github.com/sourcegraph/sourcegraph/issues/6666